server/api/attendance/attendance.controller.js

```diff
@@ -185,15 +186,15 @@ exports.verifyFullAttendance = function(req,res){
     return getPresent(userId, isoDateToTime(new Date()), classYear._id, function(err,userAttendance){
       if (err) {return handleError(err)}
       if (!userAttendance.length){
-        return res.send(400, 'No attendance found');
+        return res.status(400).json('No attendance found');
```

Comment:
It returns properly. If you use .body it returns the string in the res.text field instead